cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -1482,34 +1548,23 @@ def test_login_nodes_pool_information(
         config_mock.return_value.scheduling.settings.scheduler_definition.metadata = ""
         config_mock.return_value.scheduling.scheduler = "slurm"
 
-        # TODO Update once multiple pools supported in describe-cluster API response
-        mocker.patch.object(PoolStatus, "_retrieve_data")
-        pool_status_1 = PoolStatus("clustername", "pool1")
-        pool_status_dict = {"pool1": pool_status_1}
-
+        # Mock the LoginNodesStatus
         mocker.patch("pcluster.models.login_nodes_status.LoginNodesStatus.retrieve_data")
-        mocker.patch(
-            "pcluster.models.login_nodes_status.LoginNodesStatus.get_pool_status_dict", return_value=pool_status_dict
-        )
+        mocker.patch("pcluster.models.login_nodes_status.PoolStatus._retrieve_data")
         mocker.patch(
             "pcluster.models.login_nodes_status.LoginNodesStatus.get_login_nodes_pool_available",
             return_value=login_nodes_pool_available,
         )
-        mocker.patch("pcluster.models.login_nodes_status.PoolStatus.get_status", return_value=status)
-        if scheme:
```

Comment:
How come we are not doing these checks anymore?

Reply:
I moved this portion to [this function](https://github.com/cjmakin/aws-parallelcluster/blob/develop/cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py#L60-L76), and changed it to mock the function return values of specific objects. That way I could test different values for pool1 and pool2 in the response content